apps/users-api/src/index.ts

```diff
@@ -1,8 +1,19 @@
 import express from "express";
 import 'dotenv/config';
 import { Pool } from 'pg';
+import { z } from 'zod';
+import { parseRequest } from 'utils';
+
+const User$ = z.object({
+    id: z.number(),
```

Comment:
Je vais chier juste pcq on peut vraiment eviter des probleme en definisant bien les type
```suggestion
    id: z.number().positive(),
```

Reply:
ouais tu fait chier